taskd_services/http/api.py

```diff
@@ -58,10 +59,13 @@ def create_new(cls, org_name, user_name, **params):
         taskd_user_key = key_proc_output[0].split(":")[1].strip()
         result = key_proc.wait()
         if result != 0:
-            raise TaskdError()
+            raise TaskdError(f'command: {command}, output: {key_proc_output}')
 
         params.update(
-            {"user_key": taskd_user_key, "org_name": org_name, "user_name": user_name,}
+            {
+                "user_key": taskd_user_key,
+                "org_name": org_name, "user_name": user_name,
```

Comment:
Some of the formatting in this file is a little surprising; like most Python projects these days, this project uses `black` as the standard style.  Forgive me for neglecting to wire that up in CI.

Reply:
To be honest I'm not used to python style guidelines and I just relied on the Neovim LSP python integration.  It complained a lot about line lengths and I formated it by using the Neovim built-in format support.  After your comment, I installed `black` but when I try to format it, there are a lot of changes and this piece of code it formated in the same way as I committed.  How should I format the code to be compliant with black?

Reply:
Yeah -- I bet you're right, I'm not sure I actually ever did convert Inthe.AM to black (but did most of my other active projects).  I'll sort out the style nits.